src/XIVLauncher.Core/Program.cs

```diff
@@ -276,7 +277,7 @@ public static void CreateCompatToolsInstance()
     {
         var wineLogFile = new FileInfo(Path.Combine(storage.GetFolder("logs").FullName, "wine.log"));
         var winePrefix = storage.GetFolder("wineprefix");
-        var wineSettings = new WineSettings(Config.WineStartupType, Config.WineBinaryPath, Config.WineDebugVars, wineLogFile, winePrefix, Config.ESyncEnabled, Config.FSyncEnabled);
+        var wineSettings = new WineSettings(Config.WineStartupType, Config.WineBinaryPath, Config.DxvkHudCustomString, Config.WineDebugVars, wineLogFile, winePrefix, Config.ESyncEnabled, Config.FSyncEnabled);
```

Comment:
Probably a better idea to not put dxvk related config into `WineSettings`, especially considering since we just pass `DxvkHudType` and `DxvkAsyncEnabled` into the CompatibilityTools ctor currently.

Ideally DxvkHudCustomString is in there as well, or even better just put all dxvk related settings into their own class like `DxvkSettings` ,

Reply:
Also consider that dxvk can also be used on windows, even though this is currently not in the scope of what XL provides it is possible, and for example an option in the GShade windows installer

Reply:
Using dxvk on Windows is something we should never support. (It also will crash dalamud)

It honestly led to a support nightmare when the gshade installer started providing it as an option. 

DXVK options should only be available for Wine platforms.